eisbuk-admin/src/components/atoms/CustomerList/CustomerList.tsx

```diff
@@ -23,14 +23,14 @@ interface Props {
   customers?: Customer[];
   extended?: boolean;
   onCustomerClick?: (customer: Customer) => void;
-  className?: string;
+  classNames?: { rootDiv?: string; tableContainer?: string };
```

Comment:
Right...this is generally a bad practice to go about it this way:
- you shouldn't let the outside component (explicitly) control too much of the internals of a component 
- the convention of passing the `className` as a string should always be respected, and `className` should (in most cases) be applied to the root container of the component (in this case the outermost div element)
- in edgy edge case, you might want to do it and you should consider two things:
1. Leave the `className` as is (string applied to the root element) and additional `tableContainerClassName` (MaterialUI does this for some components)
2. As @silviot says it: put a 5$ in a jar each time you do it

In this case, I would actually go with the non-optimal option and pass them temporarily as `className` and `tableContainerClassName`, but add `/** @TODO fix scrolling */` because this is still a temporary solution (which I've added for the time being) that doesn't work for all cases (small screens) adn should be revisited.